alacritty/src/config/mod.rs

```diff
@@ -312,6 +312,14 @@ fn print_deprecation_warnings(config: &Config) {
             "Config dynamic_title is deprecated; please use window.dynamic_title instead",
         )
     }
+
+    #[cfg(windows)]
+    if config.winpty_backend {
```

Comment:
This warning is kind of useless when it's built without winpty backend? I think we should mention in that case, that winpty backend is disabled and conpty is being used, and if they want winpty, they should compile stuff manually.

Reply:
I mean the goal is for them to delete the config option, does it matter if it is enabled or not?

If it's built without the WinPTY backend, there should still be a warning that tells the user that they should delete the option?

Reply:
I'm not sure if I'd always warn tbqh, since on some systems it's desired to use it, that being sad, I'd only warn if winpty feature is enabled, and this option is there.

Reply:
I mean even if it's desired to use it, people should migrate away from it. If it's gonna be removed in two months, they're going to have to accept it anyways.

Warning even though the WinPTY feature isn't enabled allows us to tell users to actually clean up their config file, then we can just remove the variable when the feature is removed. Otherwise we'd have to warn then, which seems suboptimal?

Reply:
I guess...

Reply:
I mean if anything, shouldn't it be the other way around? Warn only when it was built **without** the WinPTY backend? If people intentionally opt into the feature then they probably know what they're doing and are aware that the feature is deprecated.

That could make some sense I think.

Reply:
Ah, yeah, the other way around, but it really doesn't matter, knowing that we'll drop it in like 2 month.